src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,86 @@
+.product-card {
+  box-sizing: border-box;
+  height: 406px;
+  width: 200px;
+  padding: 206px 16px 16px;
+  background-image: url(../images/imac.jpeg);
+  background-size: 160px 134px;
+  background-repeat: no-repeat;
+  background-position: 19px 32px;
+  border: 1px solid#f3f3f3;
```

Comment:
It's better to use shorthand properties whenever possible for consistency and brevity. Merge `border-width`, `border-style`, and `border-color` into one `border` property.